sdks/python/apache_beam/dataframe/pandas_docs_test.py

```diff
@@ -74,22 +85,56 @@ def main():
         if any(filter in path for filter in filters):
           paths.append(path)
 
+  # Using a global here is a bit hacky, but avoids pickling issues when used
+  # with multiprocessing.
+  global parallelism
```

Comment:
Is this needed because parallelism is used in run_tests? Instead of branching on parallelism inside run_tests could we just make a different method for the parallel vs single-threaded cases?

Reply:
Sure.